core/Console/Commands/Serve.php

```diff
@@ -6,6 +6,11 @@
 
 class Serve extends Command {
     public function handle(): void {
+        if (isset($this->args[0])) {
+            exec('cd ' . Application::$ROOT_DIR . "/{$this->args[0]}" . ' && php -S localhost:8080');
```

Comment:
this could use more context instead of blindly using element of an array without context

Reply:
Sure me 7 minutes ago